.github/workflows/i18n-crowdin-download.yml

```diff
@@ -113,3 +113,18 @@ jobs:
 
         env:
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+
+      - name: Get pull request number
+        if: steps.crowdin-download.outputs.pull_request_url
+        shell: bash
+        run: |
+          pr_number=$(gh pr view ${{ steps.crowdin-download.outputs.pull_request_url }} --json number -q .number)
+          echo "PULL_REQUEST_NUMBER=$pr_number" >> "$GITHUB_ENV"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I believe actually you can use `steps.crowdin-download.outputs.pull_request_number` now, since https://github.com/crowdin/github-action/pull/216 🎉

Reply:
Let's try it!